db/migrate/20160406195810_add_id_primary_key_to_join_tables.rb

```diff
@@ -49,7 +49,7 @@ def down
 
   def delete_remote_region_rows(table)
     model = Class.new(ActiveRecord::Base) { self.table_name = table }
-    col = model.column_names_symbols.first
+    col = model.column_names.first
```

Comment:
I did this because columns_names_symbols is a ManageIQ extension patch, and the usage below shouldn't care about string or symbol, so I'm using the built-in Rails method instead.